x-pack/test_serverless/functional/test_suites/security/ftr/one_discover/context_awareness/cell_renderer.ts

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import kbnRison from '@kbn/rison';
+import expect from '@kbn/expect';
+import path from 'path';
+import { FtrProviderContext } from '../../../../../ftr_provider_context';
+import { SECURITY_ES_ARCHIVES_DIR } from '../../../constants';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const PageObjects = getPageObjects(['common', 'timePicker', 'discover', 'svlCommonPage']);
+  const testSubjects = getService('testSubjects');
+  const dataViews = getService('dataViews');
+  const esArchiver = getService('esArchiver');
+
+  describe('security root profile', () => {
+    before(async () => {
+      await PageObjects.svlCommonPage.loginAsAdmin();
+      await esArchiver.loadIfNeeded(path.join(SECURITY_ES_ARCHIVES_DIR, 'auditbeat_single'));
+    });
+
+    after(async () => {
+      await esArchiver.unload(path.join(SECURITY_ES_ARCHIVES_DIR, 'auditbeat_single'));
+    });
+
+    describe('cell renderers', () => {
+      describe('host.name', () => {
+        describe('DataView mode', () => {
+          it('should open host.name flyout', async () => {
+            await PageObjects.common.navigateToActualUrl('discover', undefined, {
+              ensureCurrentUrl: false,
+            });
+            await dataViews.createFromSearchBar({
+              name: 'auditbeat-2022',
+              adHoc: true,
+              hasTimeField: true,
+            });
+            await PageObjects.discover.waitUntilSearchingHasFinished();
+            await PageObjects.discover.dragFieldToTable('host.name');
+            expect((await PageObjects.discover.getColumnHeaders()).join(', ')).to.be(
+              '@timestamp, host.name'
+            );
+            // security host.name button
+            const hostName = await testSubjects.findAll('host-details-button', 2500);
+            expect(hostName).to.have.length(1);
+            await hostName[0].click();
```

Comment:
I like seeing the click followed by verifications as you've done here, but I wonder if index access will ever lead to a brittle test?

I see the expectation is set to length === 1, which makes sense, but is it always one on real serverless (MKI)?
  
I imagine I'm barking up the wrong tree here but I'm aware of instances where there are very few guarantees on MKI.

Reply:
Thanks for your review.

Well, it should always be 1 because it depends on the data loaded on line 23.

Index `auditbeat-2022` loads only one item so that the result is predictable. I understand that might change if someone uploads more data with new index `auditbeat-2022-.....` but then this test will ofcourse will need to be update.


What i mean is that result should always be determinate. Does that answer your question?

Reply:
Makes sense for sure.  

I think the situation you mentioned will definitely be an issue in the future, but we can cross that bridge when we get there.

Reply:
Sounds like a plan! :)

Reply:
done here: https://github.com/elastic/kibana/pull/199818/commits/f608d0b6723f36515c06716d9f889da6c867757f